docs/collect-forms.rst

```diff
@@ -48,6 +50,44 @@ The :guilabel:`Drafts` list shows all drafts by the name that the :ref:`form def
   :alt: The Drafts screen. Several drafts are listed by name.
   :class: device-screen-vertical
 
+.. _finalizing_drafts:
+
+Finalizing drafts
+==================
+
+.. note::
+
+  In Collect versions prior to v2023.3, it was possible to edit finalized forms. If your workflow involves new information that can be discovered at any time, consider configuring Collect to hide the :guilabel:`Finalize` or :guilabel:`Send` button from the form end screen (see the form entry access control section of :ref:`protected settings <admin-settings>`) and using :ref:`bulk finalization <bulk-finalizing-drafts>`.
+
+If your device is online and Collect is configured to automatically send submissions, you can send send a filled form immediately from the form end screen by tapping the :guilabel:`Send` button. If your device is offline or Collect is not configured to automatically send submissions, you will need to mark a draft as finalized before it can be sent. Finalized forms are available from the :guilabel:`Ready to send` screen where they can be viewed or sent. You can finalize a form from the end screen by tapping the :guilabel:`Finalize` button.
+
+.. _bulk-finalizing-drafts:
+
+Bulk finalizing drafts
+-----------------------
+
+.. warning::
+
+  Bulk finalization does not work with submission encryption.
```

Comment:
Add a link to submission encryption.  Also, do we really call it that? It might get confused with SSL.